.github/workflows/disallow-zenhub-link.yml

```diff
@@ -0,0 +1,16 @@
+name: "Disallow Zenhub Links"
+on:
+  pull_request:
+    types:  [opened, edited, reopened, synchronize]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Disallow zenhub links
+      uses: JJ/github-pr-contains-action@releases/v2
+      with:
+        github-token: ${{github.token}}
+        bodyDoesNotContain: "zenhub.com"
```

Comment:
We can check for any multiple of value like `zenhub.com|delete this|TODO` so maybe the name of this workflow is too specific. 

Like instead of "Disallow Zenhub Links" it would be "Disallowed PR values" or something. I just wanted the failed check to be clear about why it was failing so I named it "Disallow Zenhub Links".